pkg/peer/messages.go

```diff
@@ -40,19 +46,6 @@ type DataChannelMessage struct {
 
 type DataChannelAvailable struct{}
 
-type RTCPReceived struct {
-	TrackID string
-	Packets []RTCPPacket
-}
-
-type RTCPPacket struct {
-	Type    RTCPPacketType
-	Content rtcp.Packet
+type KeyFrameRequestReceived struct {
+	common.TrackInfo
```

Comment:
Is this a PLI, should it be named that way? What about FIRs?

Reply:
So, since RTCPs [failed a lot](https://github.com/matrix-org/waterfall/issues/87) and resulted in errors, I decided that why don't we just simplify it then by converting all read RTCPs into just some "keyframe request" (so both PLIs and FIRs are just sent as keyframe request messages from the `Peer` so that it's up to the peer to then convert the "keyframe request" in what it thinks makes sense. Currently, all "keyframe requests" are indeed converted into PLIs upon arrival to the concrete peer.

This solved all the RTCP errors that were sent (so it seems like these were indeed FIRs that we tried to forward [might require more work to be properly handled]). Though the freezes in Firefox (and only in Firefox) still happen from time to time, but there are no grey frames anymore 🙂